app/services/singInService.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const crypt = require('bcrypt');
+
+let user;
+let singin;
+const routerserv = express.Router;
```

Comment:
check the bootstrap. You declare your routes in the routes file, and there you send your methods to the controller.